CMakeLists.txt

```diff
@@ -4,18 +4,22 @@
 cmake_minimum_required(VERSION 3.23)
 
 project(
-  beman.exemplar # CMake Project Name, which is also the name of the top-level
-                 # targets (e.g., library, executable, etc.).
+  beman.exemplar
   DESCRIPTION "A Beman library exemplar"
+  VERSION 0.0.0
   LANGUAGES CXX)
 
+include(CMakePackageConfigHelpers)
 include(CTest)
 include(FetchContent)
+include(GNUInstallDirs)
+
+set(TARGETS_EXPORT_NAME ${CMAKE_PROJECT_NAME}Targets)
```

Comment:
I'd rather just inline this below instead of making a variable that's used only once.